android/filament-android/src/main/java/com/google/android/filament/View.java

```diff
@@ -713,6 +713,11 @@ public void setShadowsEnabled(boolean enabled) {
      * SwapChain associated with the engine.
      * </p>
      *
+     * <p>
+     * A view with a custom render target cannot rely on Renderer.ClearOptions, which only applies
```

Comment:
That's a bit weird API-wise :/

Reply:
Well, I think it makes sense actually.

The ClearOptions clear the renderer -- not the view. When the view brings it own render target, it's just because it's an intermediate draw, that will ultimately be rendered (e.g. as a texture) into another "prime" view.

I think it would be weird for the Renderer.ClearOptions to clear some other render target.

We have a mechanism to clear a View now, and that's Skybox.